modules/tensor_mechanics/src/materials/EshelbyTensor.C

```diff
@@ -14,7 +14,9 @@ InputParameters
 validParams<EshelbyTensor>()
 {
   InputParameters params = validParams<Material>();
-  params.addClassDescription("Stuff");
+  params.addClassDescription("Computes the Eshelby tensor as a function of "
```

Comment:
Oh no!  I had a bad feeling that I was responsible for the original version, and that was just confirmed by 'git blame'.